src/app/components/typography.tsx

```diff
@@ -97,19 +97,35 @@ const titleStyles = (as: Headings) => {
   }
 };
 
-export const Title = forwardRefWithAs<BoxProps, Headings>((props, ref) => (
-  <BaseText
-    userSelect="none"
-    letterSpacing="-0.01em"
-    fontFamily="'Open Sauce One'"
-    fontWeight="500"
-    color={color('text-title')}
-    ref={ref}
-    display="block"
-    css={titleStyles(props.as)}
-    {...props}
-  />
-));
+interface TitleProps extends BoxProps {
+  /** Removes ::before pseudoelement inserted by capsize */
+  removeBeforePseudoElement?: boolean;
+}
+export const Title = forwardRefWithAs<TitleProps, Headings>((props, ref) => {
+  // This is to prevent tall graphemes (like emojis, some chinese characters,
+  // and others) getting cut off by pseudoelements inserted by capsize, which is
+  // due for removal:
+  // https://github.com/hirosystems/stacks-wallet-web/issues/2344
+  const styleObject = props.removeBeforePseudoElement
+    ? { ...titleStyles(props.as), ['::before']: undefined }
+    : titleStyles(props.as);
+  // Avoids passing this prop down to components not expecting it.
+  const { removeBeforePseudoElement: _omit, ...restProps } = props;
+
+  return (
+    <BaseText
+      userSelect="none"
+      letterSpacing="-0.01em"
+      fontFamily="'Open Sauce One'"
+      fontWeight="500"
+      color={color('text-title')}
+      ref={ref}
+      display="block"
+      css={styleObject}
+      {...restProps}
+    />
+  );
+});
```

Comment:
Nice approach Edu. 

As @fbwoolf said on Discord, we should prioritise removing capsize, as this solution adds complexity to remove a feature we want to kill anyway.

A point on comments: The philosophy we've tried to follow is to add as many as necessary, but as few as possible (or less is more). 
The first comment, with explanation and link to issue is helpful, it adds relevant information not inferable from the code. 
The second, explains what the code does. I'd argue this is already clear from the prop name. To make it even clearer, it could be a method `removeBeforePseudoElement(props) {}`, at this point, the comment doesn't add any value (and gives the next developer more to read)